app/models/doi.rb

```diff
@@ -1808,8 +1813,47 @@ def consortium_id
     client.provider.consortium_id.downcase if client.present? && client.provider.consortium_id.present?
   end
 
+  def related_dmp_ids
+    related_identifiers.select { |related_identifier|
+      related_identifier["relatedIdentifierType"] == "DOI"
+    }.select { |related_identifier|
+      related_identifier.fetch("resourceTypeGeneral", nil) == "OutputManagmentPlan"
```

Comment:
Think there is a typo here, per the [Schema XSD for ResourceTypes](https://schema.datacite.org/meta/kernel-4.4/include/datacite-resourceType-v4.xsd) it should be `OutputManagementPlan`

Reply:
This should be resolved